### ISSUE TITLE
KDE Plasma 6 - Elisa Player (needs a symbolic tray icon) usages

### ISSUE THREAD

**Issue by niri-san** [bikass/kora#190]
Elisa Player on KDE Plasma 6 is using an colored icon on tray not a symbolic ones.

Elisa Player > Application Menu > Configure Elisa > Keep running in System Tray when main window is closed

This provide tray icon on Elisa player, i see you already have an "elisa-symbolic.svg" that needs an edit to be made in white on a black panel. Also, i'm not sure why didn't reach the tray? perhaps there is using another name?

However, there is a need for correction.

Here how it appears on tray.
![Screenshot_20240530_093211](https://github.com/bikass/kora/assets/160327855/50eae966-0e49-426d-abaa-da827caddb15)

**Comment by bikass** [comment]
I could not find elisa-symbolic.svg in theme. Could you check if it works with this name, and I will make an icon.

**Comment by niri-san** [comment]
I found out that "elisa-symbolic' comes from Breeze Inherits=breeze,Adwaita,hicolor that kora is using, therefore what i see from seleting icon from KDE it is part of the breeze when the package is missing X icon.

I tried it making a symbolic at kora/actions/symbolic/elisa-symbolic.svg
![Screenshot_20240607_114246](https://github.com/bikass/kora/assets/160327855/d3ae7ca0-0ba7-4134-8070-7c22d5d0e555)
It works, it's enough to make an elisa-symbolic, but I'm not sure about the location. It worked for me "kora/actions/symbolic".
